### ISSUE TITLE
Meta-Issue: Rework Contact Blocking Mechanism

### ISSUE THREAD

**Issue by Ancanus** [emesene/emesene#249]
Currently, it is very easy to misclick the block button in the conversation window. Since there is no indication that the contact was blocked, and it is removed from your contact list, the contact could pretty well fade forever. I propose the following changes:

1) Add a confirmation box whenever you click the Block contact button.
2) Blocked contacts should not be removed from your contact list. Otherwise, it is impossible to unblock them using the contact list. (I have Order by Group, Group Offline Contacts, Show Offline Contacts, Sort by Name in case this is a bug)
3) Perhaps add a way to manage contacts in the Block List (and Allow List) in the preferences?

**Comment by c10ud** [comment]
1) not sure i want to do it (unix style)
2) only if you choose "hide blocked contacts", but then again they're hidden, not removed
3) yes, but then it's another request :D

we added an overlay in the contact's picture (thanks to andreastagi), i think it should be enough?

**Comment by astagi** [comment]
Yeah I think is enough too...Anyway I added a new feature in my last commit: when a contact is blocked, right clicking on it you will be able to do only the "unblock" operation, otherwise the "block" operation..

**Comment by c10ud** [comment]
ok, if someone has some other idea, please open a new issue
